README.md

```diff
@@ -101,10 +101,10 @@ dokku apps:create ckan-solr
 sudo docker volume create --name ckan-solr
 dokku docker-options:add ckan-solr run,deploy --volume ckan-solr:/opt/solr/server/solr/ckan
 
-sudo docker pull openafrica/solr:latest
-sudo docker tag openafrica/solr:latest dokku/ckan-solr:latest
+sudo docker pull codeforafrica/ckan-solr:2.7.6
+sudo docker tag codeforafrica/ckan-solr:2.7.6 dokku/ckan-solr:latest
```

Comment:
Looked around and don't seem to have access to the `openafrica` account. Will keep on checking but if we're switching org back to `codeforafrica`, shouldn't we rename `ckan` part to `openafrica` then?